src/plugins/discover/public/application/components/sidebar/discover_field_details.tsx

```diff
@@ -41,62 +41,66 @@ export function DiscoverFieldDetails({
-        </EuiText>
-      )}
-      {details.error && <EuiText size="xs">{details.error}</EuiText>}
-      {!details.error && (
-        <div style={{ marginTop: '4px' }}>
-          {details.buckets.map((bucket: Bucket, idx: number) => (
-            <DiscoverFieldBucket
-              key={`bucket${idx}`}
-              bucket={bucket}
-              field={field}
-              onAddFilter={onAddFilter}
-            />
-          ))}
-        </div>
-      )}
+    <>
+      <div className="dscFieldDetails">
+        {details.error && <EuiText size="xs">{details.error}</EuiText>}
+        {!details.error && (
+          <div style={{ marginTop: '4px' }}>
+            {details.buckets.map((bucket: Bucket, idx: number) => (
+              <DiscoverFieldBucket
+                key={`bucket${idx}`}
+                bucket={bucket}
+                field={field}
+                onAddFilter={onAddFilter}
+              />
+            ))}
+          </div>
+        )}
 
-      {details.visualizeUrl && (
-        <>
-          <EuiLink
-            onClick={() => {
-              getServices().core.application.navigateToApp(details.visualizeUrl.app, {
-                path: details.visualizeUrl.path,
-              });
-            }}
-            className="kuiButton kuiButton--secondary kuiButton--small kuiVerticalRhythmSmall"
-            data-test-subj={`fieldVisualize-${field.name}`}
-          >
-            <FormattedMessage
-              id="discover.fieldChooser.detailViews.visualizeLinkText"
-              defaultMessage="Visualize"
-            />
+        {details.visualizeUrl && (
+          <>
+            <EuiLink
+              onClick={() => {
+                getServices().core.application.navigateToApp(details.visualizeUrl.app, {
+                  path: details.visualizeUrl.path,
+                });
+              }}
+              className="kuiButton kuiButton--secondary kuiButton--small kuiVerticalRhythmSmall"
+              data-test-subj={`fieldVisualize-${field.name}`}
+            >
+              <FormattedMessage
+                id="discover.fieldChooser.detailViews.visualizeLinkText"
+                defaultMessage="Visualize"
+              />
+            </EuiLink>
             {warnings.length > 0 && (
               <EuiIconTip type="alert" color="warning" content={warnings.join(' ')} />
             )}
-          </EuiLink>
-        </>
-      )}
-    </div>
+          </>
+        )}
+      </div>
+      <EuiPopoverFooter>
+        {!details.error && (
+          <EuiText size="xs" textAlign="center">
+            <FormattedMessage
+              id="discover.fieldChooser.detailViews.topValuesInRecordsDescription"
+              defaultMessage="Top 5 values in"
+            />{' '}
+            {!indexPattern.metaFields.includes(field.name) && !field.scripted ? (
+              <EuiLink onClick={() => onAddFilter('_exists_', field.name, '+')}>
+                {details.exists}
+              </EuiLink>
```

Comment:
I'm very confused by what this link does.

Reply:
I'm also not sure what that link does. @kertal do you know?

Reply:
it creates an exists filter, I don't know if anybody used it because it's quite invisible. however in the last testing session somebody found it and used it. so it's displaying the number of records that have this field. and if you click on it, you get those records, maybe a tooltip here would be useful ... or a more obvious solution?

Reply:
Can you two work together to maybe find a better solution? I don't think using this number as a link is right because to me it looks like it should be working like pagination.

Reply:
Came up with this proposal. @cchaos what do you think?

<img width="1135" alt="Group 3" src="https://user-images.githubusercontent.com/4016496/89462811-d58f0000-d722-11ea-9424-ed216bccbbfe.png">

Reply:
Good catch on reducing the duplicate "Top values" text. I still don't think this clears up the question "Why is the number 74 blue, and what is going to happen when I click on it?" 

I would make the behavior of this link a completely separate link from this number. What it's doing is adding filter to the global search bar with the name of the field. For example, this is what gets created when clicking that 74:

<img width="424" alt="Screen Shot 2020-08-05 at 17 26 52 PM" src="https://user-images.githubusercontent.com/549577/89465929-f1090380-d740-11ea-98d0-529e3913a006.png">

So it's similar to those +/- buttons, but instead of filtering on those particular **values** of the field, it just filters down the results to only those documents that **the field exists in**. So basically it **is** narrowing down to those 74 documents.

Or even something like `<Link>Exists in 74</Link> / 91 records` with a tooltip explaining that it will add a filter to only those documents that contain the field.

Reply:
I think we can go with this for now
<img width="533" alt="no0tooltip" src="https://user-images.githubusercontent.com/4016496/89475867-04ff3600-d73e-11ea-81ed-ddb3cc767764.png">

There's no tooltip in it... I tried adding a tooltip but it always shows up when the popover opens and I don't think that's ideal.
![tooltip](https://user-images.githubusercontent.com/4016496/89475844-f3b62980-d73d-11ea-9a0d-6ab021a5d270.gif)

Reply:
@andreadelrio I like this solution, for me it's much clearer this way